react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/FontAssetManager.kt

```diff
@@ -0,0 +1,53 @@
+package com.revenuecat.purchases.react.ui
+
+import android.content.res.AssetManager
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import com.facebook.infer.annotation.Nullsafe
+
+@Nullsafe(Nullsafe.Mode.LOCAL)
+object FontAssetManager {
+    private val mFontCache: MutableMap<String?, List<Font>?> = hashMapOf()
+    private val EXTENSIONS = arrayOf("", "_bold", "_italic", "_bold_italic")
+    private val FILE_EXTENSIONS = arrayOf(".ttf", ".otf")
+
+    private fun getStyleFromExtension(style: Int): FontStyle {
+        return when (style) {
+            2, 3 -> FontStyle.Italic
```

Comment:
Instead of hardcoding the indexes here, could we change the `EXTENSIONS` to be an enum with a string property being the extension? Like:
```
private enum class Extensions(val extension: String) {
    REGULAR(""),
    BOLD("_bold"),
    ITALIC("_italic"),
    BOLD_ITALIC("_bold_italic");

    val weight: FontWeight
        get() = when (this) {
            REGULAR -> FontWeight.Normal
            BOLD -> FontWeight.Bold
            ITALIC -> FontWeight.Normal
            BOLD_ITALIC -> FontWeight.Bold
        }
    
    val style: FontStyle
        get() = when (this) {
            REGULAR -> FontStyle.Normal
            BOLD -> FontStyle.Normal
            ITALIC -> FontStyle.Italic
            BOLD_ITALIC -> FontStyle.Italic
        }
}
```

Reply:
hmm I see, before we needed the indexes to use it for SparseArray, but we no longer need to. Thanks I'll be changing this to an enum!